src/common/exception/message.cpp

```diff
@@ -21,8 +21,14 @@ std::string ExceptionMessage::invalidPKType(const std::string& type) {
         type);
 }
 
+std::string ExceptionMessage::overLargeStringPKValueException(uint64_t length) {
+    return stringFormat("The maximum length of primary key strings is 4096 bytes. The input "
+                        "string's length was {}.",
+        length);
+}
 std::string ExceptionMessage::overLargeStringValueException(uint64_t length) {
-    return stringFormat("Maximum length of strings is 4096. Input string's length is {}.", length);
+    return stringFormat(
+        "The maximum length of strings is 262144 bytes. The input string's length was {}.", length);
```

Comment:
maybe just say 256KB? so it's more readable.

Reply:
I was thinking it's more comparable to the length, which we're showing in bytes. It might not be very easy to compare, for example, 262145 bytes to 256KB at a glance, but if we list the exact number of bytes you can do so more easily.